.github/workflows/build-citus-community-nightlies.yml

```diff
@@ -25,7 +25,6 @@ jobs:
         platform:
           - el/7
           - el/8
-          - ol/7
```

Comment:
why we're removing those ?

Reply:
ol/7 packages does not exist in PGDG for now so we could not bake an image